src/AzureAuth/CommandMain.cs

```diff
@@ -262,10 +262,45 @@ private int GetToken()
             try
             {
                 ITokenFetcher tokenFetcher = this.TokenFetcher();
-                TokenResult tokenResult = (this.tokenFetcherOptions.Scopes == null
-                    ? tokenFetcher.GetAccessTokenAsync(this.CombinedAuthMode)
-                    : tokenFetcher.GetAccessTokenAsync(this.tokenFetcherOptions.Scopes, this.CombinedAuthMode))
-                    .Result;
+                TokenResult tokenResult = null;
+
+                // When running multiple AzureAuth processes with the same tenant and resource ID,

```

Comment:
Don't forget to mention the `this.Client` as well.
```suggestion
                // When running multiple AzureAuth processes with the same resource, client, and tenant IDs,

```